app/src/main/java/se/sigmaconnectivity/blescanner/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,59 @@
+package se.sigmaconnectivity.blescanner.ui.home
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.Observer
+import com.google.android.material.snackbar.Snackbar
+import org.koin.androidx.viewmodel.ext.android.viewModel
+import se.sigmaconnectivity.blescanner.R
+import se.sigmaconnectivity.blescanner.databinding.FragmentHomeBinding
+import se.sigmaconnectivity.blescanner.domain.feature.FeatureStatus
+import se.sigmaconnectivity.blescanner.service.BleScanService
+import se.sigmaconnectivity.blescanner.ui.base.BaseFragment
+
+class HomeFragment : BaseFragment() {
```

Comment:
nie zmieniaj nic, tu wydaje mi się, że to pasuje. Te fragmenty mają dużo wspólnego i moim zdaniem to dziedziczenie jest uzasadnione, ale niedawno czytałem:
https://proandroiddev.com/say-no-to-baseactivity-and-basefragment-83b156ed8998

No może ewentualnie zamiast własnie nazwy baseFragment moglibyśmy użyć jakiejś bardziej opisującej ich cechę wspólną.